app/assets/javascripts/angular/components/organizationOverviewSurveyResponses/organizationOverviewSurveyResponses.component.js

```
@@ -16,9 +16,25 @@ function organizationOverviewSurveyResponsesController(
     $scope,
     surveyService,
     periodService,
+    $uibModal,
+    localStorageService,
+    $log,
 ) {
+    let surveyResponseModalClosed = async modal => {
+        try {
```

Is this try needed? If there's an error it should* bubble up and get logged by rollbar.

I've also had issues with async/await and Angular 1.x. It seems to like to swallow promise rejections and never call the catch. Maybe that was only when using the api service... Were you having issues with the $log here? I think it might have something to do with not calling $digest after a promise completes like the $q implementation does.

---

yah your right, i left it there, I was testing and debuging :S